### PR TITLE
Collabo branch

### PR DESCRIPTION
**Pull request by mwanawabangona**
This project is inspired by Figma templates, I have remade a Figma template and invited my programming partners to come to see if there is any change needed on the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+</p>
+
+<section class="forms">
+<form action="#" method="post">
+  <input type="text" id="fname" name="firstname" maxlength="30" placeholder="Ellis Ng'ona" required><br><br>
+  <input type="email" id="email" name="_replyto" placeholder="ellisallday@gmail.com" required><br><br>
+  <textarea name="messaage" id="subject" maxlength="500" placeholder="write your messaage here" required></textarea>
+  <input type="submit" id="bttn" value="Get in touch">
+</form>
+</section>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Please make sure you are calling your CSS properly, Look at the image below for more details, it is displaying nothing.
form3
- [ ] Form uses the action and method attribute to integrate with [formspree](https://formspree.io/html) service.

         * action value should be similar to https://formspree.io/f/{form_id}
         * method value equals POST

File: index.html [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+<section class="ecli">
+  <img src="../image/contactcircle.png" alt="icons">
+</section>
+
+<p class="form-text">Contact me</p>
+<p class="form-paragraph">If you have application you are interested in developing,<br> a feature that you need built or a project
+  that needs coding, <br>i'd love to help with it.
+</p>
```
Comment (Emmyn5600) [inline index.html]: - [ ] Use h1 or h2 tag for Contact me instead of p tag

File: README.md [hunk 3]
```diff
@@ -1 +1,77 @@
...
+
+
+- set up a new repository and prepare it for development using best practices.
+- build mobile website.
+ 
+
+## Live Demo
+
+[Live Demo Link](https://http://127.0.0.1:5500/index.html/)
+
```
Comment (Emmyn5600) [inline README.md]: - [ ] Kindly, add the live link demo which working properly.
livelink

File: index.html [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+                          </ul>
+                        </div>
+
+                        <a href="#" class="btn-one">See project</a>
+                      </li>
+                    </ul>
+                  </div>
+               </li>
+          </ul>
+  </section>
```
Comment (Emmyn5600) [inline index.html:192]: - [ ] Please add hover color to the `see project` button and add some padding between the project technology.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @mwanawabangona 

Good job so far!:clap::clap:
You have done well in implementing the project requirements :clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

##  Required Changes:writing_hand:
  
**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,461 @@
...
+  bottom: 118em;
+  left: 25em;
+}
+
+.forms {
+  position: relative;
+  bottom: 145em;
+  left: 35em;
+  height: 50px;
+}
```
Comment (hemantso) [inline css/style.css]: - [ ] There is too much of width which leads to horizontal scrolling of the page. Kindly try to fix the positioning issue of the form section.

image

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @mwanawabangona , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]

**Comment by Emmyn5600** [comment]
Hi @mwanawabangona and the next code reviewer, I approved the changes by mistake on the GitHub review changes. Please ignore the above message saying  `Emmyn5600 approved these changes`

**Comment by mwanawabangona** [comment]
ok

**Review by Shaher-11 — APPROVED** [review]
#### Hello @mwanawabangona,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**